.github/workflows/test.yaml

```diff
@@ -268,12 +268,12 @@ jobs:
             rules: 'main'
           - group: ubuntu-no-lts
```

Comment:
Nitpick, can you rename this to `ubuntu-non-lts` when you have a chance to update the required tests in sync?